CLN v23.08 install fails during compile `File config.vars not found`

Trying to run: `config.scripts/cl.update.sh reckless` to upgrade to the lastest CLN version (23.08)

Results in this failure during compile:

```
# Building from source Core Lightning 
- Configuring EXPERIMENTAL_FEATURES enabled

Unknown option '--enable-experimental-features'
Usage: ./configure [--reconfigure] [setting=value] [options]
If --reconfigure is specified, config.vars will set defaults.

[...]

 Building Core lightning from source

File config.vars not found: you must run ./configure before running make.
make: *** No rule to make target 'config.vars', needed by 'show-flags'.  Stop.

HEAD is now at 98ad46795 CHANGELOG.md: update for 23.08 release final.
- Install to /usr/local/bin/
File config.vars not found: you must run ./configure before running make.
mkdir -p /bin
mkdir -p /libexec/c-lightning
mkdir -p /libexec/c-lightning/plugins
mkdir -p /share/man/man1
mkdir -p /share/man/man5
mkdir -p /share/man/man7
mkdir -p /share/man/man8
mkdir -p /share/doc/c-lightning
make: *** No rule to make target 'config.vars', needed by 'ccan/config.h'.  Stop.

```

@openoms I get the same in branch `4127-updatecln` when I try to install cln with the new new version v23.08 .. here is a bit longer output:

```
  • Installing pyln-testing (23.05 /home/bitcoin/lightning/contrib/pyln-testing)
- Configuring EXPERIMENTAL_FEATURES enabled

Unknown option '--enable-experimental-features'
Usage: ./configure [--reconfigure] [setting=value] [options]
If --reconfigure is specified, config.vars will set defaults.
Default settings:
CSANFLAGS =
  CC (default cc)
  CWARNFLAGS (default -Wall -Wundef -Wmissing-prototypes -Wmissing-declarations -Wstrict-prototypes -Wold-style-definition -Werror -Wno-maybe-uninitialized -Wshadow=local)
  COPTFLAGS (default -Og)
  CDEBUGFLAGS (default -std=gnu11 -g -fstack-protector-strong)
  CONFIGURATOR_CC (default cc)
    To override compile line for configurator itself
  PYTEST (default python3 -m pytest)
  VALGRIND (default 0)
Options include:
  --prefix= (default /usr/local)
    Prefix for make install
  --enable/disable-developer (default disable)
    Developer mode, good for testing
  --enable/disable-debug (default disable)
    Extra debug checks in the build, good for testing
  --enable/disable-compat (default enable)
    Compatibility mode, good to disable to see if your software breaks
  --enable/disable-valgrind (default (autodetect))
    Run tests with Valgrind
  --enable/disable-static (default disable)
    Static link sqlite3 and zlib libraries
  --enable/disable-coverage (default disable)
    Compile with Clang coverage instrumentation
  --enable/disable-address-sanitizer (default disable)
    Compile with address-sanitizer
  --enable/disable-ub-sanitizer (default disable)
    Compile with undefined behaviour sanitizer
  --enable/disable-fuzzing (default disable)
    Compile with fuzzing
  --enable/disable-rust (default disable)
    Compile with Rust support

- Building Core lightning from source

File config.vars not found: you must run ./configure before running make.
make: *** No rule to make target 'config.vars', needed by 'show-flags'.  Stop.

HEAD is now at 98ad46795 CHANGELOG.md: update for 23.08 release final.
- Install to /usr/local/bin/
File config.vars not found: you must run ./configure before running make.
mkdir -p /bin
mkdir -p /libexec/c-lightning
mkdir -p /libexec/c-lightning/plugins
mkdir -p /share/man/man1
mkdir -p /share/man/man5
mkdir -p /share/man/man7
mkdir -p /share/man/man8
mkdir -p /share/doc/c-lightning
make: *** No rule to make target 'config.vars', needed by 'ccan/config.h'.  Stop.
```

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -3,7 +3,7 @@
 - Update: RaspiOS base image from 2023-05-03
 - Update: Bitcoin Core v25.0.0 [details](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-25.0.md)
 - Update: LND v0.16.4-beta [details](https://github.com/lightningnetwork/lnd/releases/tag/v0.16.4-beta)
-- Update: Core Lightning v23.05.2 [details](https://github.com/ElementsProject/lightning/releases/tag/v23.05.2)
+- Update: Core Lightning v23.08 [details](https://github.com/ElementsProject/lightning/releases/tag/v23.08)
 - Update: Suez - Channel Visualization for LND & CLN [details](https://github.com/prusnak/suez)
 - Update: Electrum Server in Rust (electrs) v0.10.0 [details](https://github.com/romanz/electrs/blob/master/RELEASE-NOTES.md#0100-jul-22-2023)
 - Update: C-lightningREST v0.10.5 [details](https://github.com/Ride-The-Lightning/c-lightning-REST/releases/tag/v0.10.5)

--- a/home.admin/config.scripts/cl.install.sh
+++ b/home.admin/config.scripts/cl.install.sh
@@ -58,7 +58,7 @@ function installDependencies() {
 function buildAndInstallCLbinaries() {
   echo "- Configuring EXPERIMENTAL_FEATURES enabled"
   echo
-  sudo -u bitcoin ./configure --enable-experimental-features
+  sudo -u bitcoin ./configure
   echo
   echo "- Building Core lightning from source"
   echo

--- a/home.admin/config.scripts/cl.install.sh
+++ b/home.admin/config.scripts/cl.install.sh
@@ -2,7 +2,7 @@
 # https://lightning.readthedocs.io/
 
 # https://github.com/ElementsProject/lightning/releases
-CLVERSION="v23.05.2"
+CLVERSION="v23.08"
 
 # install the latest master by using the last commit id
 # https://github.com/ElementsProject/lightning/commit/master

